frontend/src/main/web/src/components/Notification.js

```diff
@@ -0,0 +1,95 @@
+import React, { Component, PropTypes } from 'react'
+import { connect } from 'react-redux'
+import {
+  ButtonRound,
+  Icon,
+  Modal,
+  Row
+} from './'
+import {
+  clearMessage
+} from '../actions/common'
+import { isEmptyOrNull } from '../utils/StringUtils'
+
+class Notification extends Component {
+
+  getSeverityClass (severity) {
+    switch (severity) {
+      case 'warn':
+        return 'C(danger)'
+      case 'error':
+        return 'C(danger)'
```

Comment:
Are these supposed to return the same thing? If they are, it can just use flow-through:

```
case 'warn':
case 'error':
  return 'C(danger)'
```